client/js/main.js

```diff
@@ -247,29 +247,34 @@ onLoad(function() {
 
   on('#muteButton', 'click', function(){
     if(muted) {
-      document.getElementById('volume').value = unmuteVol;
-      document.getElementById('muteButton').classList.remove('muted');
-      var allAudios = document.querySelectorAll('audio');
-      allAudios.forEach(function(audio){
+      $('#volume').value = unmuteVol;
+      $('#muteButton').classList.remove('muted');
+      $a('audio').forEach(function(audio){
```

Comment:
I'm not familiar with the use of `$a`

Reply:
`$` gives you the first matching element; `$a` gives you a collection of all matching elements (basically just a shorthand for `document.querySelectorAll`.